cloudburst/server/executor/server.py

```diff
@@ -273,6 +275,9 @@ def executor(ip, mgmt_ip, schedulers, thread_id):
                     else:
                         triggers = list(received_triggers[key].values())
 
+                    if fname not in function_cache:
+                        logging.error('%s not in function cache', fname)
```

Comment:
See above about returning error.

Reply:
see my comment above.

Reply:
fixed

Reply:
Can we move this into a helper function called something like `generate_error_response`? Instead of repeating it in 3 different places.